### PR TITLE
Add Manifold Markets embeds

### PR DESCRIPTION
**Pull request by sipec**
_Directly_ cribbed from OWID previews https://github.com/ForumMagnum/ForumMagnum/pull/4728

Uh, I haven't tested this yet cus I'm not a lightcone dev and thus don't have the credentials to spin up the local instance. Either someone else needs to take it from here, or I'll need to get unblocked on setup.

### CODE REVIEW

File: public/lesswrong-editor/src/ckeditor.js [hunk 1]
```diff
@@ -181,7 +181,16 @@ const embedConfig = {
 			`
 		},
 		{
-			name: 'OWID',
+			name: "Manifold",
+			url: /^manifold\.markets\/(?:embed\/)?(\w+\/[\w-]+)$/,
+			html: ([match, longslug]) => `
+				<div data-manifold-id="${longslug}" class="manifold-preview">
```
Comment (sipec) [inline public/lesswrong-editor/src/ckeditor.js:187]: I'm actually not sure what all these data id custom attributes are for!
Unless there's lesswrong-specfic reasons the other embeds have it, the iframe alone should suffice

**Comment by jpaddison3** [comment]
> Uh, I haven't tested this yet cus I'm not a lightcone dev and thus don't have the credentials to spin up the local instance. Either someone else needs to take it from here, or I'll need to get unblocked on setup.

You can run the instance with a local database! I coincidentally just made an update to the readme that should make that clearer. No credentials required. Let me know if you do think I'm wrong — I know some things like search won't work, but you should be able to test this just fine.

**Comment by Xodarap** [comment]
Thanks for submitting this!

I would update the size of the embed (or change how you are populating the iframe on your end):

image

also remember to run `yarn rebuild-reinstall-ckeditor` and commit the build files.

**Review by Xodarap — CHANGES_REQUESTED** [review]
Please update the UI and commit the build files

**Comment by sipec** [comment]
> Please update the UI

I've updated it now to be like the styling for the other embeds, but I still won't be able to test how it actually looks in context

**Comment by sipec** [comment]
how long is the mongodb local setup supposed to take? it's been several hours and still inching along...

**Comment by Xodarap** [comment]
This is still looks weird to me:

image


I think the Mongo set up should not take that long. Did it complete for you? I'm not sure what would be taking a long time – Mongo is slow, but it's not that slow.

**Comment by sipec** [comment]
It did complete, I think now, but it's quite difficult to get the client to connect to the local server correctly. It keeps disconnecting

**Review by Xodarap — APPROVED** [review]

**Comment by Xodarap** [comment]
@sipec Thanks for doing this! The change has now been deployed to the EA Forum. Want to make an announcement post? You could continue the proud tradition of copying OWID and do something similar to their post: https://forum.effectivealtruism.org/posts/PSmsGwk2kZbBXZgHi/new-forum-feature-preview-and-embed-our-world-in-data-charts

I'm not sure when LW will merge in your changes, but I would guess it's fairly soon.

**Comment by jaysonvirissimo** [comment]
Oh heck yeah!

**Pull request merged**